packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -63,6 +63,8 @@ export const Components = {
   PanelInspector: {
     Data: {
       content: 'Panel inspector Data content',
+      applyTransforms: 'Apply transforms to data',
+      applyFieldOverrides: 'Apply field overrides to data',
```

Comment:
nit: Are these buttons? Maybe follow the naming convention as we have for the `actionButton`.